backend/dataall/modules/shares_base/api/mutations.py

```diff
@@ -35,6 +40,13 @@
     type=gql.Boolean,
 )
 
+cancelShareExtensionObject = gql.MutationField(
```

Comment:
IMHO, `ShareExtensionObject` is a confusing naming.
We do not introduce another type of object, just change the properties of existing `ShareObject`.
Would you mind to call it here and in other mutations e.g. `_operation_ShareExtension`

Reply:
sure. I can do that.